modules/openapi-generator/src/test/java/org/openapitools/codegen/python/PythonLegacyClientCodegenTest.java

```diff
@@ -93,7 +93,9 @@ public void testRegularExpressionOpenAPISchemaVersion3() {
         Assert.assertEquals(op.allParams.get(5).pattern, "/^pattern\\d{3}$/i");
         // pattern_with_backslash_after_bracket '/^[\pattern\d{3}$/i'
         // added to test fix for issue #6675
-        Assert.assertEquals(op.allParams.get(6).pattern, "/^[\\pattern\\d{3}$/i");
+        // removed because "/^[\\pattern\\d{3}$/i" is invalid regex
+        // Assert.assertEquals(op.allParams.get(6).pattern, "/^[\\pattern\\d{3}$/i");
+        
```

Comment:
What about this regex is invalid?

Reply:
Right - the `[` character needs a closing bracket (`]`) in order for this to be valid.

Reply:
If we wanted the literal `[` character, we would need to escape it in the regex

Reply:
Can you add that info in the comment? 
`// removed because "/^[\\pattern\\d{3}$/i" is invalid regex because [ is not escaped and there is no closing ]`

Reply:
Done!